packages/@aws-cdk/aws-batch/test/compute-environment.test.ts

```diff
@@ -82,6 +82,164 @@ describe('Batch Compute Evironment', () => {
       });
     });
   });
+  describe('using fargate resources', () => {
```

Comment:
We also need an integ test. Add a compute environment to https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-batch/test/integ.batch.ts#L30